internal/checks/blockheight.go

```diff
@@ -24,6 +24,7 @@ type BlockHeightCheck struct {
 	logger              *zap.Logger
 	blockHeight         uint64
 	useWSForBlockHeight bool
+	ShouldRun           bool // TODO(polsar): Add config parameter to control whether this check should run.
```

Comment:
curious what's the motivation behind this? will this set to false by default? we want to run this check for all chains/upstreams